internal/engine/wazevo/call_engine.go

```diff
@@ -82,6 +82,8 @@ type (
 		memmoveAddress uintptr
 		// framePointerBeforeGoCall holds the frame pointer before calling a Go function. Note: only used in amd64.
 		framePointerBeforeGoCall uintptr
+		// memoryWaitAddress holds the address of memory wait trampoline function.
+		memoryWaitAddress *byte
```

Comment:
memoryWaitTrampolineAddress (memmove isn't many trampoline hence it doesn't have it in the name)